tests/repository_data/uptane_director_snapshot.json

```diff
@@ -0,0 +1,26 @@
+{
```

Comment:
Is this valid metadata?  Not all metadata is listed in the snapshot metadata, at least as named on disk.  For example, why don't I see `targets_simpler.json` listed anywhere?